src/Biconomy.js

```diff
@@ -1073,15 +1111,14 @@ async function callDefaultProvider(engine, payload, callback, errorMessage) {
 }
 
 
-function _getEIP712ForwardMessageToSign(request, forwarder) {
+function _getEIP712ForwardMessageToSign(request, forwarder, domainData) {
   // Update the verifyingContract field of domain data based on the current request
   if(!forwarderDomainType || !forwardRequestType || !forwarderDomainData || !forwarder || !forwarderDomainDetails) {
```

Comment:
empty forwarderDomainDetails object will still be truthy value.

Reply:
Fixed